Emby.Server.Implementations/Diagnostics/CommonProcess.cs

```diff
@@ -42,14 +43,16 @@ public CommonProcess(ProcessOptions options)
                 StartInfo = startInfo
             };
 
-            if (options.EnableRaisingEvents)
-            {
-                _process.EnableRaisingEvents = true;
-                _process.Exited += _process_Exited;
-            }
+
+            _process.EnableRaisingEvents = true;
```

Comment:
I don't think this is the right solution. We probably still want it to be configurable.

It should be changed here in options instead: https://github.com/jellyfin/jellyfin/blob/master/MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs#L449

Reply:
I see, guess I did an oopsie again. Fixed now.